README.md

```diff
@@ -4,7 +4,7 @@ RPC server and client to run the Stone Prover on the Madara sequencer.
 
 ## Description
 
-This project provides a server that can run any set of Cairo programs on top of the Starknet OS to generate the state diff and proof of execution of the programs.
+This project provides a server that can run any set of Cairo programs on top of the Bootloader to generate the state diff and proof of execution of the programs.
```

Comment:
Parts of the sentence don't make sense if you only talk about the bootloader. I suggest the following change:

```suggestion
This project provides a server that can run any set of Cairo programs on top of the Starknet bootloader and generate the proof of execution of these programs.
```